packaging/windows/OpenRA.nsi

```diff
@@ -59,7 +59,7 @@ Section "-Reg" Reg
 	; Replay file association
 	WriteRegStr HKLM "Software\Classes\.orarep" "" "OpenRA_replay"
 	WriteRegStr HKLM "Software\Classes\OpenRA_replay\DefaultIcon" "" "$INSTDIR\OpenRA.ico,0"
-	WriteRegStr HKLM "Software\Classes\OpenRA_replay\Shell\Open\Command" "" "$INSTDIR\OpenRA.exe Launch.Replay=%1"
+	WriteRegStr HKLM "Software\Classes\OpenRA_replay\Shell\Open\Command" "" "$INSTDIR\OpenRA.exe Launch.Replay='%1'"

```

Comment:
I was running this via the debugger because I'm lazy - perhaps the single quotes act differently when launching a replay properly. I'll remove my changes requested for the moment.